### DISCUSSION TITLE
Turning off password storage

### DISCUSSION THREAD

**Discussion by autay27** [nianeyna/ao3downloader#115] [Q&A]
Thank you for your tool.

Can I change the settings so it doesn't save my login details? Or is there a place in the files to manually delete it?

**Comment by nianeyna** [comment]
You can't turn off password saving *yet*, but that is a planned feature. You can track issue #12 to be notified when it goes live, if you like. In the meantime, you can manually delete the "settings.json" file to remove the saved password (though this will also delete any other settings you have saved, like your preferred download file types).
